### PR TITLE
Add example for AWS Lambda & Serverless Framework

### PR DESCRIPTION
**Pull request by connorads**
This is to add a examples of integrating Sentry with [AWS Lambda in Node and Python with Lambda Layers using Serverless Framework](https://sentry-docs-git-fork-connorads-patch-1.sentry.dev/product/integrations/cloud-monitoring/aws-lambda/#serverless-framework).

Follow up from https://github.com/getsentry/sentry-docs/pull/3830

Wasn't really sure how much detail to add with the README/docs and wasn't sure about the folder structure.
Feedback and changes welcome 🤗

### Node

image

### Python

image

### Sentry

image

### CODE REVIEW

**Review by kamilogorek — APPROVED** [review]
LGTM! Feel free to merge it @ahmedetefy if you also +1

**Comment by ahmedetefy** [comment]
Hey @connorads 

Thanks for opening the PR. The Serverless examples look good

However, rather than changing the directory stucture, I think it makes more sense to just add a `serverless-framework` dir to this path for Node `aws-lambda/node` that contains your serverless example 
and well the same for Python as well!

**Review by ahmedetefy — CHANGES_REQUESTED** [review]
Hey @connorads 

Thanks for opening the PR. The Serverless examples look good

However, rather than changing the directory stucture, I think it makes more sense to just add a `serverless-framework` directory to the following path `aws-lambda/node` for Node that contains your Serverless example, and well the same for Python as well!

**Comment by connorads** [comment]
Hey @ahmedetefy, let me know what you think of the new structure

**Comment by connorads** [comment]
Hey @ahmedetefy, we good to merge this now?

File: aws-lambda/node/serverless-framework/serverless.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    SENTRY_TRACES_SAMPLE_RATE: '1.0'
+    SENTRY_DSN: ${env:SENTRY_DSN}
+    NODE_OPTIONS: '-r @sentry/serverless/dist/awslambda-auto'
+
+plugins:
+  - serverless-latest-layer-version
+
+custom:
+  layers:
+    - arn:aws:lambda:${self:provider.region}:943013980633:layer:SentryNodeServerlessSDK:latest
```
Comment (connorads) [inline aws-lambda/node/serverless-framework/serverless.yml:19]: Oops, note to self: Need to change this back to remove the `latest` stuff - this was some testing for https://github.com/getsentry/sentry-javascript/issues/3834 that slipped in here when doing the folder moves 😬

Comment (connorads) [reply aws-lambda/node/serverless-framework/serverless.yml]: Fixed in https://github.com/getsentry/examples/pull/101/commits/cbc427cd781e65b7eb436b9bd54ddf49f6b9703d

**Comment by kamilogorek** [comment]
Thanks!

**Pull request merged**